src/libs/elektra/backend.c

```diff
@@ -231,7 +231,13 @@ Backend * backendOpen (KeySet * elektraConfig, KeySet * modules, KeySet * global
 			else
 			{
 				// no one cares about that config
-				if (!failure) ELEKTRA_ADD_WARNING (16, errorKey, keyBaseName (cur));
+				if (!failure)
+					// TODO: Correct? (INTERNAL)
+					ELEKTRA_ADD_INSTALLATION_WARNINGF (
```

Comment:
This is indeed tricky. Can you give a concrete example? Does such an error come from a specification or how exactly is this triggered?
Since it was already present before elektra started I guess installation does fit more properly here.

Reply:
In general such errors (validation errors during kdbGet()) can easily happen if someone tampers with config files.

For this specific warning, you do not even need to "tamper" because no validation happens during kdbSet():
```
kdb mount tests.ecf /tests
kdb set system/elektra/mountpoints/\\/tests/something
kdb ls /tests
 Sorry, 1 warning was issued ;(
        Sorry, module  issued the warning 16:
        unrecognised Config Tree: something
/tests/import/test
```

But even with a safe `kdb set` such warnings are not avoidable: users can always change underlying data.

Reply:
Ah I see, then I guess syntactic validation fits most appropriate when the underlying data has not the expected format/ entries anymore that elektra expects it to have?

Reply:
Yes, to call it validation makes sense. Although keys that are present that should not be is for me "structural validation".